test/elm/comparison/data.coffee

```diff
@@ -51,8 +53,32 @@ define EqRatios: 10 'mg' : 2 'dL' = 15 'mg' : 3 'dL'
 define UneqRatios: 10 'mg' : 2 'dL' = 15 'mg' : 4 'dL'
 ###
 
+###
+Translation Error(s):
+[11:28, 11:42] Could not resolve call to operator Equal with signature (tuple{Foo:System.Any},tuple{Bar:System.Any}).
+[15:40, 15:94] Could not resolve call to operator Equal with signature (tuple{a:System.String,b:System.Any},tuple{a:System.String,c:System.Any}).
```

Comment:
Have you looked into the errors above? Are these valid test cases that point out an issue in CQL-to-ELM? Or are they invalid test cases? If the latter, should they be fixed or removed?